src/login/logind-utmp.c

```diff
@@ -135,11 +135,11 @@ int manager_setup_wall_message_timer(Manager *m) {
                 return 0;
         }
 
-        if (elapse < n)
+        if (elapse && elapse < n)
```

Comment:
Please use `if (elapse > 0 && elepse < n)`. See CODING_STYLE, we use "boolean operations" only for booleans and pionters (and sometimes for convenience, but only in limited circumstances).

Reply:
ack